webapp/channels/src/components/drafts/drafts.tsx

```diff
@@ -56,41 +129,68 @@ function Drafts({
             <Header
                 level={2}
                 className='Drafts__header'
-                heading={formatMessage({
-                    id: 'drafts.heading',
-                    defaultMessage: 'Drafts',
-                })}
+                heading={(
+                    <FormattedMessage
+                        id='drafts.heading'
+                        defaultMessage='Drafts'
+                    />
+                )}
```

Comment:
Not a big deal, but this sends a new `heading` prop on every render to the `Header` component. Two solutions are reverting this change (so it is always a string, that is comparable) or moving the definition of this component out of the component (so it is always the same reference to the same object).

But 0/5. It is a good practice to be "as stable as possible", but probably the inner component is not memoizing anything on that prop either way.